server/src/internalClusterTest/java/org/elasticsearch/indices/IndicesLifecycleListenerIT.java

```diff
@@ -122,10 +120,13 @@ public void beforeIndexCreated(Index index, Settings indexSettings) {
             });
         clusterAdmin().prepareReroute().add(new MoveAllocationCommand("index1", 0, node1, node2)).get();
         ensureGreen("index1");
-        ClusterState state = clusterAdmin().prepareState().get().getState();
-        List<ShardRouting> shard = RoutingNodesHelper.shardsWithState(state.getRoutingNodes(), ShardRoutingState.STARTED);
-        assertThat(shard, hasSize(1));
```

Comment:
This has failed according to the build scan.
Adding a logging so that we know if the shard was still relocating for some reason or if it become unassigned